<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVPStudio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/resources.qrc">
    <normaloff>:/icons/icons/icon256.ico</normaloff>:/icons/icons/icon256.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNewContent"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionOpenSource"/>
    <addaction name="action_report"/>
    <addaction name="actionCheckUpdate"/>
    <addaction name="separator"/>
    <addaction name="action_help"/>
    <addaction name="action_follow"/>
    <addaction name="action_donate"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionWavGenerator"/>
    <addaction name="actionImageOrganizer"/>
    <addaction name="actionMXLPlayer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_T"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNewContent">
   <property name="text">
    <string>新建播放内容</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于AVPStudio...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionWavGenerator">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../res/resources.qrc">
     <normaloff>:/icons/icons/wavgenerator_icon256.ico</normaloff>:/icons/icons/wavgenerator_icon256.ico</iconset>
   </property>
   <property name="text">
    <string>WAV生成器</string>
   </property>
  </action>
  <action name="actionImageOrganizer">
   <property name="icon">
    <iconset resource="../../../res/resources.qrc">
     <normaloff>:/icons/icons/imageorganizer_icon256.ico</normaloff>:/icons/icons/imageorganizer_icon256.ico</iconset>
   </property>
   <property name="text">
    <string>图片排列器</string>
   </property>
  </action>
  <action name="actionMXLPlayer">
   <property name="icon">
    <iconset resource="../../../res/resources.qrc">
     <normaloff>:/icons/icons/mxlplayer_icon256.ico</normaloff>:/icons/icons/mxlplayer_icon256.ico</iconset>
   </property>
   <property name="text">
    <string>MXL播放器</string>
   </property>
  </action>
  <action name="actionOpenSource">
   <property name="text">
    <string>开放源代码</string>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="text">
    <string>检查更新...</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助...</string>
   </property>
  </action>
  <action name="action_report">
   <property name="text">
    <string>报告问题</string>
   </property>
  </action>
  <action name="action_follow">
   <property name="text">
    <string>关注与支持</string>
   </property>
  </action>
  <action name="action_donate">
   <property name="text">
    <string>捐赠</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
